tests/unit/v2/RedisClient.js

```diff
@@ -0,0 +1,21 @@
+const RedisClient = require('../../../libV2/redis');
+
+describe('Test RedisClient', () => {
+    let client;
+
+    beforeEach(() => {
+        client = new RedisClient({});
+        client.connect();
+    });
+
+    afterEach(() => client.disconnect());
```

Comment:
Does this work? I am not sure if this results in some weird race condition.
```suggestion
    afterEach(client.disconnect);
```

Reply:
Not sure I get what you mean. I guess we could but we would need to bind to the object instance by appending `.bind(client)`. I copied this logic from another test suite without thinking too much about it. It seems to handle both callback and async syntax, so the way it is I think works by returning a promise to `afterEach`.

Reply:
ok yea, I still gotta get used to bind.